app/pages/settings/email.jsx

```diff
@@ -200,8 +204,25 @@ class EmailSettingsPage extends React.Component {
     });
   }
 
-  requestConfirmationEmail() {
-    console.log('+++ Send confirmation email!')
+  async requestConfirmationEmail() {
+    try {
+      this.setState({ requestConfirmationEmailStatus: 'posting' });
+      const url = `${apiHost}/users/confirmation`;
+      const bearerToken = await getBearerToken(authClient);
+      const res = await fetch(url, {
+        method: 'POST',
+        mode: 'cors',
+        headers: {
+          accept: 'application/vnd.api+json; version=1',
```

Comment:
```suggestion
          Accept: 'application/vnd.api+json; version=1',
```

This might be why content negotiation is failing.

Reply:
Actually, looking at the auth client, these are the headers that it sends. Note that the Accept header is just `application/json`. Your Accept header here is requesting a JSON-API resource, but I don't think the auth APIs follow the JSON-API specs.

https://github.com/zooniverse/panoptes-javascript-client/blob/8157794dfacfbc1f5d41c5730b2f47aae6fc013a/lib/config.js#L75-L78

They are exported as `config.jsonHeaders`.